Sources/SwiftExtensions/String.swift

```diff
@@ -3,4 +3,8 @@ extension String {
     public static func * (l: String, r: Int) -> String {
         repeatElement(l, count: r).joined()
     }
+    
+    public func titlecase() -> String {
+      return prefix(1).uppercased() + lowercased().dropFirst()
```

Comment:
I think string offers `capitalized` which does this for you

Reply:
Yeah, capitalisation is really complex and better left to Apple's natural language frameworks.